### ISSUE TITLE
Feature: Always set language to detected, unless manually specified

### ISSUE THREAD

**Issue by laundmo** [LucasGGamerM/moshidon#531]
**Is your feature request related to a problem? Please describe.**
When writing a reply to a post which has set the wrong langauge, it's annoying having to change it manually.
Its also annoying having to change it when writing a new post which is not in my default language.

**Describe the solution you'd like**
There already is a "detected" option when choosing post language for a post which already has some text.
I would like to allow setting the default post language to "detected". I would also like to be able to force my default language to overwrite the language of the post i'm replying to, ideally in combination with this "detected" option.

**Describe alternatives you've considered**
Just a setting to force-set my replies language instead of using the original posts language would already be plenty. This would cover 90% of cases, where i'm replying in my default language to a wrongly language-tagged post.

**Comment by maxfragg** [comment]
Agree. It feels frustrating that there is a detected entry, but that I can't use it as default.
